Game_Manual/Boundaries.md

```diff
@@ -19,18 +19,29 @@ At this point, they are ready to pursue their craft as a practitioner and no lon
 
 ## Fail Conditions
 
-A player fails the game if their [stats][stats] show them to be consistently underperforming.
+A player is at risk of failing the game if their [stats][stats] show them to be consistently underperforming.
 
-Fail conditions are triggered when any of the following persists over multiple cycles:
+There are two stages of failing.
 
-1. Rate of [XP][xp] growth is too low
-1. [Learning Support][learning-support] is too low
-1. [Culture Contribution][culture-contribution] is too low
-1. [Estimation Accuracy][estimation-accuracy] is too low
+The first stage is the _Danger Zone_. When a stat falls below any of the following thresholds, it is in the Danger Zone:
 
-If fail conditions are met, the Moderator will ask the player to leave the game.
+- [Team Play][team-play] : 80
+- [Culture Contribution][culture-contribution] : 80
+- [XP][xp] earned per cycle : 15
 
-_For now, since we have very little historical data tracking stats, "too high" and "too low" are the best we can do in defining these stat thresholds. It's up to the Moderator to decide what the thresholds are for each stat and to communicate these thresholds to the players._
+Once a stat is within Danger Zone, the player has to consistently improve their affected stats until they are above the threshold. If you have a stat in the Danger Zone, get whatever support you need to move the needle and get out of danger.
+
+The second stage is an _Exit Interview_, and is entered into when a player's stats have _both_ (a) been in the Danger Zone for at least two weeks and (b) have not shown any improvement.
+
+During the Exit Interview, the player will meet with a moderator to discuss their performance and whether or not they can continue in the game. If the moderator is not convinced that the player is putting in the right effort, they will ask them to leave the game.
```

Comment:
I'm not sure about using "right effort" as the guiding principle here. In some cases, I think someone can be putting in as much effort as possible, and just not have the skills necessary to succeed in the environment.